### ISSUE TITLE
PyScript npm module?

### ISSUE THREAD

**Issue by echarles** [pyscript/pyscript#675]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a duplication
- [X] I already searched in Google and didn't find any good information or help

### What is the issue/comment/problem?

Is there a pyscript package published in [npm](https://www.npmjs.com/)?

This would be useful as alternative to loading the script from a HTML script tag for third-party developers.

(I have searched the issues and google, but have not find that question).

**Comment by marimeireles** [comment]
Hey @echarles we hear you!
It's not super high on the priority list for now but it's definitely something in our minds for the future.
If you  or anyone experienced in creating npm packages want to give a try though, I'll give my best to offer support.
Cheers and thanks for opening the issue :)

**Comment by frog-o** [comment]
I don't want to hijack this thread but i thought  There is a pyscript in npm (unless that's different) and I tried to discuss the ability to use node package in pyscript (the opposite direction of what you are doing in the discussion and  it was closed and told me it is being tracked in this issue).
What i wanted to do is use 
https://unpkg.com/ or probably this
https://www.skypack.dev/
And have it look something like this 
<py-config>
      packages = ["matplotlib", "pandas"]
      npm-pack= ["codemirror, jquery"] 
</py-config>

I was wondering if pyscript accept something like this if i could write such a thing in a fork or if you had better ideas. I wanted to do such a thing so i could use

https://codemirror.net

Easily.
